aemsites/888de
# 192 link target fix

## Ticket/Issue number(s)
Fix  192
Fix 190

## Test URLs
- Before: https://main--888de--aemsites.hlx.page/drafts/dfink/blank-page
- After: https://192-link-target-fix--888de--aemsites.hlx.page/drafts/dfink/blank-page

## Testing instructions
- Make sure rules are honored
- When target attribute is set - external links should honor the value
- No negative side effects 
- Make sure external left-nav links open in a new window
